src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/history.html.twig

```diff
@@ -23,23 +23,23 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  *#}
 
-<table class="table">
+<table class="table" id="js-test-history-grid-table">
```

Comment:
Please let's avoid adding dedicated selectors for tests

Reply:
I mean the `js-test` prefix

Reply:
```suggestion
<table class="table" id="js-history-grid-table">
```

Agree with this then

Reply:
```suggestion
<table class="table" data-role="history-grid-table">
```

After a discussion, we agreed on this instead.